src/EnergyPlus/WaterToAirHeatPumpSimple.cc

```diff
@@ -1352,6 +1303,40 @@ namespace WaterToAirHeatPumpSimple {
 			if ( RatedCapCoolSensDes < SmallLoad ) {
 				RatedCapCoolSensDes = 0.0;
 			}
+			if ( RatedCapCoolTotalAutoSized && RatedCapCoolSensAutoSized ) {
+				if ( RatedCapCoolSensDes > RatedCapCoolTotalDes ) {
```

Comment:
I am comparing auto-sized sensible and total capacities, so both should be auto-sized. if ( RatedCapCoolSensDes > RatedCapCoolTotalDes ).